### ISSUE TITLE
Quiz results not showing

### ISSUE THREAD

**Issue by Mperce72** [fpcorso/quiz_master_next#433]
The quiz results are not showing. I have been using this wonderful plug in for a while now and it has been working great. Last night I updated my wordpress hosting and it's a newer php and now all of the sudden the quizzes are not showing any results-just a blank page. Thanks in advance for any help you can give me!

**Comment by Mperce72** [comment]
Disregard my previous post, the issue has been resolved by web hosting service. Thanks anyway and keep making great plug ins!

**Comment by fpcorso** [comment]
Hey @Mperce72 . Thank you for choosing to use this plugin! Did the host mention what the cause of the issue was so we know in case someone else encounters it?

**Comment by Mperce72** [comment]
Yes they did and it was so simple I was embarrassed. All they had to do was upgrade my wordpress to the latest version. I should have done that first, so it was a "D'oh!" moment for me.  But just in case it happens for anyone else, I upgraded to the latest php with Bluehosts optimized hosting for wordpress and it did wreck havoc with several plug ins. Thankfully all I had to do to get yours working is update my copy of wordpress. 

Thank you for your quick response, I really appreciate it. 

Michael Perce

--------------------------------------------
On Fri, 5/20/16, Frank Corso <notifications@github.com> wrote:

 Subject: Re: [fpcorso/quiz_master_next] Quiz results not showing (#433)
 To: "fpcorso/quiz_master_next" <quiz_master_next@noreply.github.com>
 Cc: "Mperce72" <sasionline1@yahoo.com>, "Mention" <mention@noreply.github.com>
 Date: Friday, May 20, 2016, 12:22 PM
 
 Hey @Mperce72 . Thank you
 for choosing to use this plugin! Did the host mention what
 the cause of the issue was so we know in case someone else
 encounters it?
 
 —
 You are receiving this because you were mentioned.
 Reply to this email directly or view
 it on GitHub
